### PR TITLE
Add Katy & Robert as maintainers

### PR DESCRIPTION
**Pull request by gonuke**
adding @katyhuff & @FlanFlanagan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,4 +103,13 @@ In order to produce a uniquely identifiable distribution:
...
+   back to 0.
+
+Feedstock Maintainers
+=====================
+
+* [@FlanFlanagan](https://github.com/FlanFlanagan/)
+* [@bam241](https://github.com/bam241/)
+* [@gonuke](https://github.com/gonuke/)
+* [@scopatz](https://github.com/scopatz/)
+* [@katyhuff](https://github.com/katyhuff/)
```
Comment (scopatz) [inline README.md]: Did you do this manually?  This gets generated by rerendering.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -44,3 +44,5 @@ extra:
     - scopatz
     - gonuke
     - bam241
+    - katyhuff
+    - FlanFlanagan
```
Comment (scopatz) [inline recipe/meta.yaml:48]: @katyhuff @FlanFlanagan are you OK with being listed as maintainers here?

**Comment by FlanFlanagan** [comment]
Yes, on all! Thanks @scopatz

File: recipe/meta.yaml (hunk 2 above)
Comment (katyhuff) [reply recipe/meta.yaml:48]: yes

File: README.md (hunk 1 above)
Comment (gonuke) [reply README.md]: yes - just pushed a revert of this

**Pull request merged**